packages/flash-calendar/src/components/Calendar.tsx

```diff
@@ -68,8 +68,25 @@ export interface CalendarProps extends UseCalendarParams {
   calendarMonthHeaderHeight?: number;
   /** Theme to customize the calendar component. */
   theme?: CalendarTheme;
+  /** Enable dark mode */
+  darkMode?: boolean;
```

Comment:
Let's make this more flexible by changing to:
```ts
  
  /**
   * When set, Flash Calendar will use this color scheme instead of the system's
   * value (`light|dark`). This is useful if your app doesn't support dark-mode,
   * for example.
   *
   * We don't advise using this prop - ideally, your app should reflect the
   * user's preferences.
   */
  colorSchemeToOverride?: ColorSchemeName;
```